components/brave_wallet/browser/zcash/zcash_rpc.h

```diff
@@ -16,40 +16,48 @@
 #include "base/memory/weak_ptr.h"
 #include "base/types/expected.h"
 #include "brave/components/brave_wallet/browser/zcash/protos/zcash_grpc_data.pb.h"
+#include "components/keyed_service/core/keyed_service.h"
 #include "components/prefs/pref_service.h"
 #include "services/network/public/cpp/shared_url_loader_factory.h"
 #include "services/network/public/cpp/simple_url_loader.h"
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
-namespace brave_wallet::zcash_rpc {
+namespace brave_wallet {
 
 // lightwalletd interface
-class ZCashRpc {
+class ZCashRpc : public KeyedService {
```

Comment:
We don't need this as a separate KeyedService as it is never going to be instantiated without everything else. I think I'll soon remove all wallets services and keep only one BraveWalletService

Reply:
I'd stay with keeping KeyedServices since, i believe, we will end in something similar but self-writen